public/assets/scss/_fonts.scss

```diff
@@ -0,0 +1,43 @@
+%page-heading-font-stack {
+    color: $blue;
+    font-weight: bold;
+    font-size: 4rem; // 100px
+}
+%title-font-stack {
+    color: $green;
+    font-size: 1.12rem; //28px
```

Comment:
Can you remove the commented `px values` beside the font size? It is a little confusing since most people assume that `1 rem` is roughly equal to 16px.